src/main/java/org/jitsi/videobridge/cc/BitrateController.java

```diff
@@ -1545,6 +1523,15 @@ int getTargetIndex()
                 ? ratedIndices[ratedTargetIdx].encoding.getIndex() : -1;
         }
 
+
+        int getPreferredIndex()
```

Comment:
can this be private or protected?

Reply:
all these methods (gettargetindex, gettargetbitrate, getpreferredindex, etc) already are package-private and intellij idea doesn't raise a warning. If we change them to be protected then intellij idea gives a warning. We can make them private (i don't see the reason why but i don't really mind either), but i think they should all be the same for concistency

Reply:
It's always good to scope methods as tightly as they can be. This better isolates the class, makes testing and future refactoring easier and better communicates how methods are supposed to be used (or not be used externally).

Reply:
Note that all these are members of a private class, so they are already only accessible from `BitrateController`